src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -89,52 +71,68 @@
     :else 0))
 
 (react/defc WorkspaceDetails
-  {:render
-   (fn [{:keys [props state refs]}]
+  {:refresh-workflow
+   (fn [{:keys [props state]}]
+     (endpoints/call-ajax-orch
+       {:endpoint (endpoints/check-bucket-read-access (:workspace-id props))
+        :on-done (fn [{:keys [success? status-code]}]
+                   (swap! state assoc :bucket-status-code status-code :bucket-access? success?))})
+     (endpoints/call-ajax-orch
+       {:endpoint (endpoints/get-workspace (:workspace-id props))
+        :on-done (fn [{:keys [success? get-parsed-response status-text]}]
+                   (if success?
+                     (swap! state assoc :workspace (get-parsed-response))
+                     (swap! state assoc :workspace-error status-text)))}))
```

Comment:
We decided not to.  It causes some issues because the workspace very briefly goes nil and then back to (usually) the same value as before.  This was causing things to flicker unnecessarily and also triggered more reloads of things.